src/Raven.Client/Documents/Operations/ETL/Queue/AzureQueueStorageConnectionSettings.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Linq;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.ETL.Queue;
+
+public sealed class AzureQueueStorageConnectionSettings
+{
+    public Authentication Authentication;
+
+    public string GetStorageUrl()
+    {
+        string storageAccountName = GetStorageAccountName();
+        return $"https://{storageAccountName}.queue.core.windows.net/";
```

Comment:
Should "https" be hard coded? For local development with the usage of emulator a developer will provide a connection string with "http://" IIRC

Reply:
I found in tests below:
> QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1

So when we use a connection string we should get the URL from it instead of building it manually I think

Reply:
So this is doable only if **ConnectionString** is used. In other cases, we need to provide service URL, which is in this format: `return $"https://{storageAccountName}.queue.core.windows.net/";`

https://learn.microsoft.com/en-us/rest/api/storageservices/addressing-queue-service-resources

Is it ok to change it only for ConnectionString then?

Reply:
Yes

Reply:
done